item-descriptions.lua

```diff
@@ -590,53 +596,54 @@ descriptions = {
+                "and must be powered for operation."},
     ORTHOPEDIC_CAST = {
                 "Casts are made from plaster, and are used to keep broken bones in",
                 "place until they are healed. Applying a cast requires a bucket,",
                 "cloth and a water source."},
     PIPE_SECTION = {
                 "An enormous piece of pipe, it is a part of a screw pump."},
     QUERN = {   "A hand-operated mill for plants, grains, and seeds. It mills plants",
-                "much slower than a millstone. Must be built from stone."},
-    QUIVER = {  "Item used to hold ammunition, made out of leather. Hunting dwarves",
-                "and crossbow dwarves will automatically grab one to store their ammo."},
-    RING = {    "A ring is an item of jewellery, which does not interfere with",
-                "wearing other equipment.  Eleven rings can be worn on each finger",
+                "much slower than a millstone."},
+    QUIVER = {  "Item used to hold ammunition, made out of leather. Hunters and archers",
+                "will automatically grab one to carry their ammo."},
+    RING = {    "A ring is an item of jewelry, which does not interfere with",
+                "wearing other equipment. Eleven rings can be worn on each finger",
                 "or toe, for a maximum of 220 rings."},
-    ROCK = {    "A small rock, sharpened as a weapon in Adventure mode."},
+    ROCK = {    "A small rock that can be knapped into tools and weapons in",
+                "Adventure Mode."},
     ROUGH = {   "Rough gemstones and raw glass are cut by a Gem Cutter at a Jeweler's",
                 "Workshop into small decorative gems. Sometimes, the gem-cutting job",
                 "results in a craft or large gem that is useless except as a very",
                 "valuable trade good."},
     SCEPTER = { "A scepter is a short, ornamental rod or wand typically associated",
-                "with royalty. It's only use is as a trade good."},
+                "with royalty. Its only use is as a trade good."},
     SKIN_TANNED = {
-                "The tanned hide of animals is flexible enough to be made into an",
+                "The tanned hides of animals are flexible enough to be made into an",
                 "assortment of goods for military and civilian use. Leather can also",
                 "be used to decorate items with sewn images at a Leather Works. Armor",
-                "and shields made from leather are not terribly effective, but are",
+                "and shields made of leather are not terribly effective, but are",
                 "still better than nothing at all."},
-    SLAB = {    "A memorial stone, used to quiet restless ghost when engraved with",
-                "the name of the deceased and built."},
+    SLAB = {    "A memorial stone, used to calm a restless ghost when engraved with",
+                "the name of the deceased and constructed."},
```

Comment:
"built" seems appropriate to me since it's the "b: Building" menu. Constructed could imply crafted at a workshop too.